.travis.yml

```diff
@@ -32,12 +32,31 @@ matrix:
       env: WP_TRAVISCI=lint
     - php: 7.3
       env: WP_TRAVISCI=test-js
+    - php: 7.3
+      env: WP_TRAVISCI=test-e2e
 
 install:
+  - set -e
```

Comment:
This [means that](https://ss64.com/bash/set.html) if a command exits with non-zero, the build will fail. For example, if `npm run env start` fails, the e2e test will fail also. 

So it makes sense to let it display the error, and the build.